package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@mitodl/smoot-design",
+  "version": "0.0.0",
+  "packageManager": "yarn@4.5.1",
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build --docs",
```

Comment:
The `--docs` is new. It omits stories and only includes the docs files.

I find https://mitodl.github.io/mit-learn a bit cluttered, so simplified some stories and thought that docs-only would be a good approach for now.

Let me know what you think.